src/test/java/org/jboss/xavier/integrations/route/MainRouteBuilder_DirectCalculateFlagSharedDisksTest.java

```diff
@@ -105,9 +126,78 @@ public void mainRouteBuilder_DirectCalculate_JSONOnVersion1_0_0Given_ShouldRetur
 
         camelContext.createProducerTemplate().sendBodyAndHeaders("direct:flags-shared-disks", body, headers);
 
+        //Then
+        Exchange result = camelContext.createProducerTemplate().send("direct:flags-shared-disks", exchange -> {
```

Comment:
Could we invoke `send` just once and hence removing the above `sendBodyAndHeaders` method invocation?

Reply:
@mrizzi Thanks for detecting this